### DISCUSSION TITLE
Fail to identify two instances when mosquitoes are mating

### DISCUSSION THREAD

**Discussion by JacopoRazzauti** [talmolab/sleap#1397] [Help!]
Hello,

I am using SLEAP to do pose estimation of 5 body parts (thorax, abdomen 1, abdomen 2, abdomen 3 and abdomen 4) along the body of two mosquitoes during mating to estimate the curvature of their abdomens. I obtained good results with a top-down (non id, although I might try id too) multianimal tracking. However, the classifier is unable to identify both instance while the mosquitoes are attached to one another (like in the second image):
![image](https://github.com/talmolab/sleap/assets/137450435/963f8da4-78f1-4d5b-88ef-93bae3a4f23d)
 
![image](https://github.com/talmolab/sleap/assets/137450435/26a0ea8b-7d0d-4b57-bc47-4be28d7102f3)

I do not know if the problem resides in the hyperparameter I have been using for training:
![image](https://github.com/talmolab/sleap/assets/137450435/d43e3cdc-eab8-47bb-bb9e-df3643e9e6b8)
![image](https://github.com/talmolab/sleap/assets/137450435/23502621-0efa-4bb0-bcbf-fced51f2a630)
Or if instead the inability to detect two instances is due to the fact that the abdo4 body parts of both mosquitoes are superimposed on top of one another during genital engagement.

I thank you in advance and happy to answer any question you might have.

**Comment by talmo** [comment]
Hi @JacopoRazzauti,

This can definitely be tricky, but we've managed this in _D. melanogaster_ videos before. It mostly just took a lot more labeling of frames where the animals are copulating.

You can adjust your model settings to optimize for this though. From your screenshot what I'd recommend is (for centered instance):
- Rotation augmentation --> -180 to 180
- Filters rate --> 1.5
- Sigma --> 2.5
- Output stride --> 2

Give that a spin and lot us know how it goes!

Cheers,

Talmo
